poc-07.py

```diff
@@ -14,12 +14,12 @@ class Message(object):
     def __init__(self, uid=None, body=None, flags=None):
         self.uid = uid
         self.body = body
-        self.flags = {'read': False, 'important': False}
+        self.flags = flags or {'read': False, 'important': False, 'delete': False}
```

Comment:
This is not done right, a wrong call can will destroy all the integrity of the message.